Lib/ufoLib/glifLib.py

```diff
@@ -297,11 +298,8 @@ def readGlyph(self, glyphName, glyphObject=None, pointPen=None):
 		text = self.getGLIF(glyphName)
 		self._purgeCachedGLIF(glyphName)
 		tree = _glifTreeFromString(text)
-		if self.ufoFormatVersion < 3:
-			formatVersions = (1,)
-		else:
-			formatVersions = (1, 2)
-		_readGlyphFromTree(tree, glyphObject, pointPen, formatVersions=formatVersions)
+		maxVersion = 1 + (self.ufoFormatVersion >= 3)
```

Comment:
nice trick summing bool with int, but IMHO it makes the code less readable for no real benefit ;)